articles/container-apps/networking-configuration.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Networking configuration in Azure Container Apps environment
+description: Learn how to configure networking in Azure Container Apps.
+services: container-apps
+author: craigshoemaker
+ms.service: azure-container-apps
+ms.topic:  conceptual
+ms.date: 10/03/2024
+ms.author: cshoe
+---
+
+# Networking configuration in Azure Container Apps environment
+
+Azure Container Apps run in the context of an environment, with its own virtual network (VNet). This VNet creates a secure boundary around your Azure Container Apps [environment](environment.md). This article tells you how to configure your VNet.
+
+## HTTP edge proxy behavior
+
+Azure Container Apps uses the [Envoy proxy](https://www.envoyproxy.io/) as an edge HTTP proxy. Transport Layer Security (TLS) is terminated on the edge and requests are routed based on their traffic splitting rules and routes traffic to the correct application.
```

Comment:
Envoy is an implementation detail we'd rather not discuss publicly, especially since we've been using other proxies for new scenarios.
```suggestion
Azure Container Apps uses an edge HTTP proxy which terminates Transport Layer Security (TLS) and routes requests to each application.
```

Reply:
Added.